apps/fm/js/fm.js

```diff
@@ -885,9 +885,8 @@ function init() {
 
 document.addEventListener('DOMLocalized', function() {
   // localize MHz
-  document.l10n.format('mhz').then(str => {
-    mhzString = str.value;
-    $('frequency').dataset.unit = mhzString;
+  document.l10n.formatValue('mhz').then(str => {
+    $('frequency').dataset.unit = mhzString = str;
```

Comment:
I brought back the explicit assignment to `mhzString` in the updated version of the pr.